src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  gap: $cellGap;
+  flex-wrap: wrap;
+  width: calendarWidth($cellSize, $weekLength, $calendarPadding, $cellGap);
+  padding: $calendarPadding;
+
+  &__cell {
+    @include square;
+    display: flex;
+    background-color: $cellColor;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition: all 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+        align-self: center;
+        margin: auto;
+        line-height: 100px;
+        font-size: 30px;
+      }
+    }
+
+    @each $day, $i in $monthStart {
```

Comment:
Consider using a more descriptive name for the loop variable. For example, `startDay` instead of `day`. 🎨